docs/webhooks/index.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Webhooks
+intro: In this section you will learn how to create an incoming webhook.
```

Comment:
Suggesting a broader/generic text that leaves room to use same structure in the future for documenting both incoming and outgoing hooks:
```suggestion
intro: In this section you will learn how to create webhooks to receive events from external systems and to notify external systems of events that triggered on your applications and infrastructure environments.
```